MapboxMobileEvents/MMEEvent.m

```diff
@@ -1,12 +1,15 @@
 #import <CommonCrypto/CommonDigest.h>
-
+#import <CoreLocation/CoreLocation.h>
 #import "MMEEvent.h"
-
+#import "MMEEventCOnfigProviding.h"
```

Comment:
This works because APFS is case-preserving but case-insensitive by default, common practice to run build machines on case-sensitive file systems to catch these issues.